packages/terra-card/src/Card.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import './Card.scss';
+
+const propTypes = {
+  /**
+   * Provides 10px on top and bottom
+   */
+  padVertical: PropTypes.bool,
+  /**
+   * Provides 10px on left and right
+   */
+  padHorizontal: PropTypes.bool,
+  /**
+   * Child Nodes
+   */
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  padVertical: true,
+  padHorizontal: true,
+};
+
+const Card = ({
+  padVertical,
+  padHorizontal,
+  children,
+  ...customProps
+}) => {
+  // Follows these conventions https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties
+  let containerClass = 'cardContainer';
+  const cardClass = classNames([
+    'card',
+    customProps.className,
+  ]);
+
+  if (padVertical) {
+    containerClass += ' veritcalPadding';
+  }
+  if (padHorizontal) {
+    containerClass += ' horizontalPadding';
+  }
+
+
+  const content = <div className={containerClass}> {children}</div>;
+
+  return (
+    <div {...customProps} className={cardClass}>
+      {content}
```

Comment:
The content wrapper div originally was because we were handling clickable cards and it was needed for something. Agreed, let's remove it and can always bring it back later if required for something